src/wobbly/storage.py

```diff
@@ -169,48 +177,43 @@ async def list_expired(self) -> list[Job]:
 
     async def list_jobs(
```

Comment:
Could the`@retry_async_transaction` decorator be useful here as well?

Reply:
I don't *think* that a pure read transaction can fail and thus shouldn't need to be retried. The write transactions need this because of the isolation level, since they do a read and then make changes based on that read and therefore need to fail if the data they read was out of date at the time of the commit. But that scenario shouldn't apply here, I think.